File: 백준/Bronze/1264. 모음의 개수/모음의 개수.py
while(True):
  count = 0
  a = input()
  if a == '#':
    break
  for i in range(len(a)):
    if 'a' == a[i] or 'A' == a[i] or 'e' == a[i] or 'E' == a[i] or 'i' == a[i] or 'I' == a[i] or 'o' == a[i] or 'O' == a[i] or 'u' == a[i] or 'U' == a[i]:
      count += 1

  print(count)

File: 백준/Bronze/1264. 모음의 개수/모음의 개수.c
#include <stdio.h>
#include <string.h>

int main(void) { 
   char string[257];
   while(1){
      fgets(string, sizeof(string), stdin);
      string[strcspn(string, "\n")] = '\0';
      if(strcmp(string, "#") == 0){
         break;
      }
      int length = strlen(string);
      int count = 0;
      for(int i = 0; i < length; i++){
         if(string[i] == 'a' || string[i] == 'e' || string[i] == 'i' || string[i] == 'o' || string[i] == 'u' || string[i] == 'A' || string[i] == 'E' || string[i] == 'I' || string[i] == 'O' || string[i] == 'U'){
            count += 1;
         }
      }
      printf("%d\n", count);

   }

   return 0;
}